core/scripts/chaincli/handler/bootstrap.go

```diff
@@ -0,0 +1,83 @@
+package handler
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/web"
+)
+
+const (
+	bootstrapJobSpec = `type = "bootstrap"
+schemaVersion = 1
+name = "ocr2keeper bootstrap node"
+externalJobID = "%s"
+id = "1"
```

Comment:
should this always be id = 1, or can we leave it blank to get auto generated?

Reply:
Done